packages/remix/src/index.ts

```diff
@@ -0,0 +1,22 @@
+import type { ActionFunctionArgs, LoaderFunctionArgs } from "@remix-run/node";
+import { createFrames as coreCreateFrames } from "frames.js/core";
+export { Button, type types } from "frames.js/core";
+
+export { fetchMetaData } from "./fetchMetaData";
+
+// @todo provide better type that can change returned handler type
+export const createFrames: typeof coreCreateFrames =
+  function createFramesForNextJS(options: any) {
```

Comment:
can we avoid using `any`?

Reply:
not at the moment, I would move that to after we finish work because I will need to change types in core and export it so I can specify everything here.

Reply:
okay - this is only used internally, right? or is it user facing

Reply:
only internally, user sees the same type as in core.